### DISCUSSION TITLE
Windows11 任务栏窗口覆盖到任务栏快捷方式图标上了

### DISCUSSION THREAD

**Discussion by legomix** [zhongyang219/TrafficMonitor#1313] [Ideas]
windows11下，设置左侧右侧都覆盖到任务栏快捷方式图标上。点选靠近任务栏窗口靠近图标而不是靠近任务栏两侧，任务栏窗口消失。这个情况是重新安装windows11后出现的。

**Comment by AnYon101** [comment]
最新版的action里作者已经修改好了，登录github下载替换就可以啦
